modules/common/react/README.md

```diff
@@ -74,7 +74,13 @@ includes custom `data-*` attributes such as `data-test-id` to help facilitate au
 
 #### `containerElement: Element`
 
-> The element that contains the portal children when `portal` is true.
+> The containing element for Popper elements. The Popper uses {@link
+> https://reactjs.org/docs/portals.html Portals} to place the DOM elements of the Popper in a
+> different place in the DOM to prevent issues with overflowed containers. When the popper is
+> opened, `aria-hidden` will be added to siblings to hide background content from assistive
```

Comment:
I know we do this manually in modal - have we confirmed that it popper does this for us? Are we sure this is what we want for all popup use cases? If a tooltip pops up, I wouldn't call all of the rest of the content on the page hidden